guides/source/documents.yaml

```
@@ -219,6 +219,11 @@
       work_in_progress: true
       url: active_support_instrumentation.html
       description: This guide explains how to use the instrumentation API inside of Active Support to measure events inside of Rails and other Ruby code.
+    -
+      name: Active Support Structured Events
+      work_in_progress: true
+      url: active_support_structured_events.html
+      description: This guide introduces Active Support Structured Events, what they are, and how to use them.
```

This description could be used for all other guides as well, just replace "Active Support Structured Events" with any other guide topic 😉 We can describe structured somewhat already.

Maybe something like:
```suggestion
      description: This guide introduces the Active Support Structured Events API for obserability.  Structured events capture named facts about what happened, when, and from where.
```